infra/base-images/base-clang/Dockerfile

```diff
@@ -25,7 +25,8 @@ RUN apt-get update && apt-get install -y wget sudo && \
     chmod +x cmake-$CMAKE_VERSION-Linux-x86_64.sh && \
     ./cmake-$CMAKE_VERSION-Linux-x86_64.sh --skip-license --prefix="/usr/local" && \
     rm cmake-$CMAKE_VERSION-Linux-x86_64.sh && \
-    SUDO_FORCE_REMOVE=yes apt-get remove --purge -y wget sudo
+    SUDO_FORCE_REMOVE=yes apt-get remove --purge -y wget sudo && \
+    rm -rf /usr/local/doc/cmake /usr/local/bin/cmake-gui /usr/local/bin/cmake
```

Comment:
LGTM with a change needed.

I think removing "/usr/local/bin/cmake" is incorrect, we want cmake binary. Rest "/usr/local/doc/cmake /usr/local/bin/cmake-gui" are nice to remove.

Reply:
Oops. My calculation of savings was still right though.